app/controllers/publishers/registrations_controller.rb

```diff
@@ -98,7 +98,15 @@ def resend_authentication_email
     private
 
     def locale_from_header
-      request.env['HTTP_ACCEPT_LANGUAGE'].scan(/^[a-z]{2}/).first == 'ja' ? :ja : :en
+      accept_language = request.env['HTTP_ACCEPT_LANGUAGE'].scan(/^[a-z]{2}/).first
```

Comment:
I think we should revert this entire block because the `HTTP_ACCEPT_LANGUAGE` will never be `jabap`.